src/test/scala/memeid/UUIDSpec.scala

```diff
@@ -146,3 +156,48 @@ class UUIDSpec extends Specification with ScalaCheck {
   }
 
 }
+
+class V1Spec extends Specification with ScalaCheck {
+  implicit val cs: ContextShift[IO] = IO.contextShift(scala.concurrent.ExecutionContext.global)
```

Comment:
You can use [`IOMatchers`](https://github.com/etorreborre/specs2/blob/c842ee48487a1c20471abc37ee56fbda8a23a18e/cats/shared/src/main/scala/org/specs2/matcher/IOMatchers.scala#L71) to provide `Clock` and `ContextShift` instances